all_files/all_functions.py

```diff
@@ -7,35 +7,35 @@
 
 
 # JUST for seeing all sports names (without any other details) and this data is not related to schedule data
-def getAllSports():
-    allSports = 'https://olypi.com/sports/?call=GetAllSports'
-    response = requests.get(allSports)
-    allSportsData = response.json()
-    return allSportsData
+def get_all_sports():
+    all_sports = 'https://olypi.com/sports/?call=GetAllSports'
+    response = requests.get(all_sports)
+    all_sports_data = response.json()
+    return all_sports_data
 
-# pp(getAllSports())
 
+# pp(get_all_sports())
```

Comment:
also this commment is not related to this line , but could you please delete the @megdye 's file as well, so when we merge your file we have everything updated.
Please run every file before assign me or anyone to your changes and check if every file is running and showing you the right answer. I want to add some new changes as well , it would be great to have your PR as quickly as possible.
Thanks melanie :)